### DISCUSSION TITLE
Exception when running SQL Profiler - Can it be ran on Azure Synapse

### DISCUSSION THREAD

**Discussion by awilsonaccenture** [microsoft/azuredatastudio#21820] [Q&A]
Getting this error:

[Error - 1:33:29 PM] Request profiler/getsessions failed.
  Message: An exception occurred while executing a Transact-SQL statement or batch.
  Code: 0 

Can the profiler be used on Azure Synapse

**Comment by erinstellato-ms** [comment]
Hi @awilsonaccenture - using Profiler with Azure Synapse is currently not supported and we do have a request open for this (https://github.com/microsoft/azuredatastudio/issues/11152).
